apps/web/pages/[user].tsx

```diff
@@ -58,6 +58,15 @@ export function UserPage(props: InferGetServerSidePropsType<typeof getServerSide
       telemetry.event(telemetryEventTypes.embedView, collectPageParameters("/[user]"));
     }
   }, [telemetry, router.asPath]); */
+
+  if (entity?.isUnpublished) {
```

Comment:
Including the unpublished screen on user pages if entity (team or org) holding the user is unpublished